tests/core/full_node/test_coin_store.py

```diff
@@ -54,39 +55,48 @@ def get_future_reward_coins(block: FullBlock) -> Tuple[Coin, Coin]:
     return pool_coin, farmer_coin
 
 
+class DBConnection:
+    async def __aenter__(self) -> DBWrapper:
+        self.db_path = Path("fndb_test.db")
```

Comment:
I know you're just refactoring, but I'd recommend using `Path(tempfile.NamedTemporaryFile().name)` here instead of a hard-coded name so we can use multiple `DBConnection` objects simultaneously.

Reply:
ah I was looking for this. I just found that `os.tempnam()` had been removed.